L72-core-google_default_credentials-extension.md

```diff
@@ -0,0 +1,62 @@
+Allow Call Credentials to be Specified in `grpc_google_default_credentials_create`
+----
+* Author(s): rbellevi
+* Approver: markdroth
+* Status: Draft
+* Implemented in: Core
+* Last updated: July 6th, 2020
+* Discussion at: (TODO)
+
+## Abstract
+
+This document proposes that `grpc_google_default_credentials_create` be
+amended to allow the user to specify their desired call credential, while
+also adding a `void* reserved` argument to lessen the burden of any such
```

Comment:
I don't see the value in adding a `void* reserved` argument.  If we were to actually use that for anything, we'd want to change the type from `void*` to something more specific anyway, so the change would still require a gRFC.

Reply:
Fair enough. Is there a reason that this pattern features so heavily in the core APIs then?

Reply:
Removed.

Reply:
I suspect that this pattern predated the decision that we are allowed to make breaking changes in the C-core API.